ractor_actors/src/net/tcp/session.rs

```diff
@@ -149,21 +169,19 @@ where
         state: &mut Self::State,
     ) -> Result<(), ActorProcessingErr> {
         match message {
-            Self::Msg::Send(msg) => {
+            Send(msg) => {
                 tracing::debug!(
-                    "SEND: {} -> {} - '{:?}'",
-                    state.local_addr,
-                    state.peer_addr,
-                    msg
+                    "SEND: {} -> {} - '{msg:?}'",
+                    state.stream_info.local_addr,
+                    state.stream_info.peer_addr
                 );
                 let _ = state.writer.cast(SessionWriterMessage::Write(msg));
             }
-            Self::Msg::FrameReady(msg) => {
+            FrameReady(msg) => {
                 tracing::debug!(
```

Comment:
ah we should probably drop this to trace level, this will be VERY noisy

Reply:
Yes. I think it is a bit too much to have them there at all, if someone want to trace the bytes they could just do it themselves. Though it is sometimes nice to see exactly the bytes that are going in/out on the wire. Perhaps it could be put in a trait that the users could mix in?

I'm updating to `trace` for now.